static/templates/admin/resultEntry.html

```diff
@@ -28,7 +28,7 @@
 			<% for (var i in metadata.tags) { 
 				var tag = metadata.tags[i];
 				%>
-				<a class="status" title="<%= tag.text %>" ><%= _.escape(tag.label) %></a>
+				<a class="status" title="<%= tag.text %>" ><%= _.escape(tag) %></a>
```

Comment:
Is `tag.text` still a thing? Seems like the tags are just strings now. It would probably be a good idea to have help text for each tag though, which could be passed in as a map. You're welcome to add that, or create a follow up ticket for it